Datagrid/ListMapper.php

```diff
@@ -125,7 +125,7 @@ public function add($name, $type = null, array $fieldDescriptionOptions = array(
         } elseif ($fieldDescription->getLabel() !== false) {
             $fieldDescription->setOption(
                 'label',
-                $this->admin->getLabelTranslatorStrategy()->getLabel($fieldDescription->getLabel(), 'list', 'label')
+                $fieldDescription->getLabel()
             );
         }
```

Comment:
`$fieldDescription->getLabel()` and `$fieldDescription->getOption('label')`
It is not the same